.gitignore

```diff
@@ -37,3 +37,6 @@ yarn.lock
 
 # Ignore local db volume
 .docker/postgres/data
+
+# Ignore certs
+certs/*
```

Comment:
This was part of an abandoned attempt to use certs for jwt signing, we can leave in or I can remove?

Reply:
Leave it for this PR... we can remove it later if we decide for sure we don't want to try to use certfiicates